cnvlib/gary.py

```diff
@@ -204,6 +205,11 @@ def autosomes(self, also=()):
 
     def by_chromosome(self):
         """Iterate over bins grouped by chromosome name."""
+
+        # necessary because of pandas bug which says:
```

Comment:
Does the pandas bug correspond to an issue in the upstream repo (pydata/pandas) that we can link to?